README.rst

```diff
@@ -17,16 +17,12 @@ An rpyc server is used in order to communicate with your computer. The interface
 Installation
 ============
 
-The process to install PyRedPitaya on the board requires the installation of Python first. See `this link <https://github.com/clade/RedPitaya/tree/master/python>`_.
+The process to install pyrp3 on the board requires the installation of Python first. See `this link <https://github.com/clade/RedPitaya/tree/master/python>`_.
```

Comment:
Why is clade's fork needed? Is it because he managed to create a workflow for a reasonably high enough version of Python?

Reply:
Completely removed python installation instructions.